README.md

```diff
@@ -4,13 +4,29 @@ This tutorial demonstrates the use of MicroProfile technologies for implementing
 made up of a number of Open Liberty Guides each of which demonstrates different MicroProfile technologies. Each Guide is 
 designed to be taken independently so if you just want to learn about a specific technology you can just take that guide. If, however, you're goal is to learn about all the MicroProfile capabilities, then working through them in the order shown below is recommended.
 
+## Tutorial Preparation
+
+The save time during the tutorial, it's best to set up your machine ahead of time. The instructions below show the pre-requisites and how to avoid lengthy downloads.
+
 ### Pre-requisites
 
 To use these guides you need the following pre-requisites:
 1. A Java 8 JDK (e.g. https://adoptopenjdk.net/?variant=openjdk8-openj9)
 1. Apache Maven (https://maven.apache.org/)
+1. A git client
 1. An editor with Java support (e.g. Eclipse, VS Code, IntelliJ)
 
+### Downloads
+
+If you would like to set up your machine ready to run through the tutorial at a location where you will have limited network connectivity, run the following commands to populate your local .m2 repo and Docker cache.
```

Comment:
This sentence suggests running this command at a location where you have limited connectivity.

Reply:
Good catch.  I'll rephrase.